src/gui/mzroll/mainwindow.cpp

```diff
@@ -226,7 +226,20 @@ using namespace mzUtils;
       _massCutoffWindow->setMassCutoffAndType(massCutoff,massCutoffType);
     */
 
-    QString clsfModelFilename = settings->value("clsfModelFilename").value<QString>();
+
+    // always load model file present in the bin directory.
+    QString clsfModelFilename;
+
+    #if defined(Q_OS_WIN) || defined(Q_OS_LINUX)
+      clsfModelFilename = QApplication::applicationDirPath() + "/" + "default.model";
```

Comment:
Use this instead 

QDir::cleanPath(QApplication::applicationDirPath() + QDir::separator()  + "default.model");

Reply:
👍 thanks for pointing out